src/main/mondrian/rolap/SqlConstraintUtils.java

```diff
@@ -696,43 +746,70 @@ public static boolean hasMultiPositionSlicer(List<Member> slicerMembers) {
         List<Member> members,
         Evaluator evaluator)
     {
-      return expandSupportedCalculatedMembers(members, evaluator, false);
+      return expandSupportedCalculatedMembers(members, evaluator, false, null);
```

Comment:
Can be just expandSupportedCalculatedMembers(members, evaluator, false), can't it?